src/app.cljs

```diff
@@ -0,0 +1,21 @@
+(ns app
+  (:require [uix.core :refer [defui $]]
+            [uix.dom]))
+
+(defui app []
+  (let [[state set-state!] (uix.core/use-state 0)]
+    ($ :<>
+       ($ :button {:on-click #(set-state! dec)} "-")
+       ($ :span state)
+       ($ :button {:on-click #(set-state! inc)} "+"))))
+
+(defonce root
+  (uix.dom/create-root (js/document.getElementById "root")))
+
+(defn render
+  []
+  (uix.dom/render-root ($ app) root))
+
+(defn ^:export init
+  []
+  (render))
```

Comment:
looks like the app won't render since `init` is not called anywhere?

Reply:
would it work in a prod build as well? 🤔

Reply:
It should. https://shadow-cljs.github.io/docs/UsersGuide.html#_modules
> :init-fn | Fully qualified symbol pointing to a function that should be called when the module is loaded initially.